R/estimate_diversity.R

```diff
@@ -128,6 +128,86 @@ divnet <-  function(W,
   }
   
   output_list[["X"]] <- X
+  
+  # Adding class alpha-estimates
```

Comment:
The construction LGTM, I actually think this should happen in the line
`output_list <- get_diversities(zz, samples_names)`
i.e. `get_diversities` should be modified, rather than overwriting it at this point in the code.

Reply:
Can you do a new pull request with this nested in `get_diversities`? @bryandmartin

Reply:
e6baa1f1102f97e18a5348c6f0a8a9886a85f324

It works now, but requires a strange workaround to get the variance